src/pages/learn/index.mdx

```diff
@@ -1,42 +1,46 @@
-> with practical tutorials, see [the available training courses](/community/resources/training-courses).
+<p className="learn-subtitle">Learn about GraphQL, how it works, and how to use it</p>
+
+GraphQL is a query language for your API, and a server-side runtime for executing queries using a type system you define for your data. The [GraphQL specification](https://spec.graphql.org/) was open-sourced in 2015 and has since been implemented in a variety of programming languages. GraphQL isn't tied to any specific database or storage engine—it is backed by your existing code and data.
 
-GraphQL is a query language for your API, and a server-side runtime for executing queries using a type system you define for your
-data. GraphQL isn't tied to any specific database or storage engine and is instead backed by your existing code and data.
+<Callout type="info">If you're already familiar with GraphQL and would like to read documentation on how to build a GraphQL service, then there are libraries to help you implement GraphQL in [many different languages](/community/tools-and-libraries/?tags=server). There are also many libraries available that allow [client applications to query existing GraphQL APIs](/community/tools-and-libraries/?tags=client).</Callout>
 
-A GraphQL service is created by defining types and fields on those types, then providing functions for each field on each type.
-For example, a GraphQL service that tells you who the logged in user is (`me`) as well as that user's name might look
-like this:
+## Describe your API with a type system
+
+A GraphQL service is created by [defining types and their fields](/learn/schema/), and then writing a function for each field to [provide the required data](/learn/execution/). For example, a GraphQL service that tells you the name of a logged-in user might look like this:
 
 ```graphql
 type Query {
   me: User
 }
 
 type User {
-  id: ID
   name: String
 }
 ```
 
 Along with functions for each field on each type:
 
 ```js
-function Query_me(request) {
-  return request.auth.user
+// Provide data for the `me` field on the `Query` type
+function Query_me(obj, args, context, info) {
+  return context.request.auth.user
 }
 
-function User_name(user) {
-  return user.getName()
+// Provide data for the `name` field on the `User` type
+function User_name(obj, args, context, info) {
+  return context.db.getUserFullName(obj.id)
 }
 ```
 
-After a GraphQL service is running (typically at a URL on a web service), it can receive GraphQL queries to validate and execute.
-The service first checks a query to ensure it only refers to the types and fields defined, and then runs the provided functions
-to produce a result.
+In the example above, the function that provides data for the `me` field on the `Query` type uses information about the authenticated user who made the request, while the the `name` field on the `User` type is populated by using that user's ID to fetch their full name from an existing database.
```

Comment:
I don't think "existing" is an important qualifier here, unless I'm missing something?

```suggestion
In the example above, the function that provides data for the `me` field on the `Query` type uses information about the authenticated user who made the request, while the the `name` field on the `User` type is populated by using that user's ID to fetch their full name from a database.
```